### ISSUE TITLE
Homework js2 week 3

### ISSUE THREAD

**Issue by mogenshoj** [kfazinic/hyf-homework#1]
Good job! It looks good, i added some comments here and there though :)
No solution for `hyfBay - get the okay'est products here`? Or did i miss it somewhere?

**warmup**
Very nice, good job! Some small things.

Watch the code syntax, it's difficult to read if the syntax is not written in a 'correct' way. (i think this is only in 1 place)

e.g.
`const delayLog = () => {setTimeout(() => console.log("Called after 2.5 seconds"), 2.5*1000);}`

should be:
```
const delayLog = () => {
  setTimeout(() => console.log("Called after 2.5 seconds"), 2.5*1000);
}
```

Also watch variable naming:
```
let d = 5;
let s = "This string logged after 5 seconds";
```
Give proper readable names, e.g. `delay` and `outputString` or something similar that is readable.

Nice that you use destructuring!
`let [lat, long] = [position.coords.latitude, position.coords.longitude]`

It doesn't give that much benefit to use it like this though, it could be:
`const { latitude, longitude } = position.coords`

Nice solution to this!
```
const doubleClick = (doubleClickedFunc) => {
    let clicked = false;
    return () => {  
        if (!clicked) {
            clicked = true;
            setTimeout(() => (clicked=false), 0.5*1000);
        } else {
            clicked = false;
            doubleClickedFunc();
        }
    };
}
```

**game**
There is a small bug somewhere, when i played the game `L` won, but the colored background showed that `S` won :)

It looks good though. Really like that you wrapped it all in a `startGame` function.
Some small comments.

This is a bit redundant:
`let [countS, countL] = [0, 0];`

would probably just write it as:
```
let countS = 0;
let countL = 0;
```

If you only want to iterate over an array and not return anything, `.forEach` would be more correct than `.map` (either one works though).
```
    displays.map(d => {
        d.innerText = null;
        d.classList.remove("winner", "loser");
        d.parentNode.classList.remove("winnerSquare");
        d.parentNode.getElementsByClassName("instructions")[0].innerText = `Press ${d.id==="displayCountOfS" ? "S" : "L"}`
    });
```

Interesting approach to finding the winner:
```
            displays.sort((a,b)=>countL-countS);
            displays[0].setAttribute("class", "winner");
            displays[0].parentNode.classList.add("winnerSquare");
            displays[1].parentNode.getElementsByClassName("instructions")[0].classList.add("loser");
```
It's fine, but can be a bit hard to follow with sorting and using indexes.
Using a condition like `if (countL > countS) { ... }` would probably be easier to read.

**Comment by kfazinic** [comment]
Thank you for the feedback, Mads! 🤠 I applied the changes.
